CHANGELOG.md

```diff
@@ -406,23 +433,27 @@ SDK Core| v0.20.7 | [symbol-sdk](https://www.npmjs.com/package/symbol-sdk)
 [v1.0.1]: https://github.com/nemfoundation/symbol-explorer/releases/tag/v1.0.1
 [v1.1.1]: https://github.com/nemfoundation/symbol-explorer/releases/tag/v1.1.1
 [v1.1.2]: https://github.com/nemfoundation/symbol-explorer/releases/tag/v1.1.2
+[v1.1.3]: https://github.com/nemfoundation/symbol-explorer/releases/tag/v1.1.3
 
 [catapult-server@v0.10.0.5]: https://github.com/nemtech/catapult-server/releases/tag/v0.10.0.5
 [catapult-server@v0.10.0.7]: https://github.com/nemtech/catapult-server/releases/tag/v0.10.0.7
 [catapult-server@v0.10.0.8]: https://github.com/nemtech/catapult-server/releases/tag/v0.10.0.8
 [catapult-server@v1.0.0.0]: https://github.com/nemtech/catapult-server/releases/tag/v1.0.0.0
 [catapult-server@v1.0.2.0]: https://github.com/symbol/catapult-client/releases/tag/v1.0.2.0
+[catapult-server@v1.0.3.0]: https://github.com/symbol/catapult-client/releases/tag/v1.0.3.1
 
 [symbol-sdk@v0.23.0]: https://www.npmjs.com/package/symbol-sdk/v/0.23.0
 [symbol-sdk@v0.23.3-alpha]: https://www.npmjs.com/package/symbol-sdk/v/0.23.3-alpha-202102181227
 [symbol-sdk@v0.23.3]: https://www.npmjs.com/package/symbol-sdk/v/0.23.3
 [symbol-sdk@v1.0.0]: https://www.npmjs.com/package/symbol-sdk/v/1.0.0
 [symbol-sdk@v1.0.1]: https://www.npmjs.com/package/symbol-sdk/v/1.0.1
 [symbol-sdk@v1.0.2]: https://www.npmjs.com/package/symbol-sdk/v/1.0.2
+[symbol-sdk@v1.0.3]: https://www.npmjs.com/package/symbol-sdk/v/1.0.3
 
 [catapult-rest@v2.3.0]: https://github.com/nemtech/catapult-rest/releases/tag/v2.3.0
 [catapult-rest@v2.3.3]: https://github.com/nemtech/catapult-rest/releases/tag/v2.3.3
 [catapult-rest@v2.3.4]: https://github.com/nemtech/catapult-rest/releases/tag/v2.3.4
 [catapult-rest@v2.3.6]: https://github.com/nemtech/catapult-rest/releases/tag/v2.3.6
 [catapult-rest@v2.3.7]: https://github.com/symbol/catapult-rest/releases/tag/v2.3.7
 [catapult-rest@v2.3.8]: https://github.com/symbol/catapult-rest/releases/tag/v2.3.8
```

Comment:
i don't think you need to show diffs for any of dependencies. just diffs for _this_ project.

Reply:
right, i will remove all of this 👍🏼